arm/Microsoft.DocumentDB/databaseAccounts/gremlinDatabases/.test/deploy.full.test.bicep

```diff
@@ -0,0 +1,32 @@
+var databaseAccountName = 'cosmosDb01'
```

Comment:
The current CI environment still only supports JSON-Parameter file. Though it pains, please adjust this and the other test files.
Also, we require a `.parameters` folder and certain conventions to build the resource name and or e.g. name the file.